vulyk/models/task_types.py

```diff
@@ -101,6 +103,30 @@ def export_reports(self, qs=None):
         for task in qs:
             yield self.answer_model.objects(task=task)
 
+    def get_leaders(self):
+        """Return sorted list of tuples (user_id, tasks_done)
```

Comment:
You've started writing docstrings in one particular format and continued with another one – isn't good to me.
My vote is for reST (https://www.python.org/dev/peps/pep-0287/) - it allows IDEs to make maximum possible type inference.